### ISSUE TITLE
Provide businesses with industry-related assessments (Business Assessment Tool)

### ISSUE THREAD

**Issue by aPruner** [Aliserag/CovidImpact#42]
### Overview:
Our tool would be easier to use if we didn't have to tell users to guess (with a very rough %age of 25, 50, or 75%) how much revenue they are losing due to covid. We should consult open data sets/research papers to see how much revenue businesses in different industries are losing because of covid. We could also provide options (in new future issues) for company size/geographic-specific assessments.

### What needs to be done:
1. Define a set of industries to research.
2. Consult open data sets/research papers and try to map industries to assumed losses for each industry (could use web scraping, grab big data sets, do it manually, consult researchers, etc), could even create a re-occurring job to pull in and consult new data.
3. With results, change the UX of the business assessment tool to add an industry dropdown to the form which maps to the resulting assumed loss in revenue. Also, allow the user to change the loss in revenue if they know how much revenue they're losing.

### Criteria of success:
1. A good set of industries to research is defined.
2. Good data science work/research is done and reasonable numbers are found, and potentially a cronjob is created to update the assumed losses in real time.
3. UX is changed to add the new values to the form, and old functionality is not lost (users can still manually enter a loss in revenue if they wish).

### Documentation:
N/A

### Stakeholder:
Who is interested in this issue resolved? It could also be the person who created this issue. Use tags (e.g @aliserag)
@aPruner

**Comment by Aliserag** [comment]
I agree we should remove the % decrease, I was talking to @NoirKomBatman the other day about potentially replacing the current Assessment Page with a 'Business Immunizer'. It'll take in inputs that will be used to return a risk score (which we can tie the industry research you highlighted above into), curated list of tips/funding/opportunities and contribute to the generation of the heatmap. Here's a mockup I was able to put together:

![Recommendations (1)](https://user-images.githubusercontent.com/11467748/81887207-75d5ee00-9553-11ea-9f27-8d989605539c.png)

what are your thoughts @aPruner?
